howto/APIs.md

```
@@ -401,6 +401,137 @@ Using the `errors.WrapWithStatus` function has the same effect as `errors.Wrap`
+
+You may want to customize this error response structure to match your API conventions, support legacy clients, or provide additional context. grpc-gateway provides the `WithErrorHandler` option to achieve this.
+
+### Custom Error Handler
+
+To customize the error response format, create a custom error handler and pass it to the `runtime.NewServeMux()`:
+
+```go
+package main
+
+import (
+	"context"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)